cg/apps/sequencing_metrics_parser/parsers/bcl2fastq.py

```diff
@@ -104,3 +109,73 @@ def get_bcl2fastq_stats_paths(demultiplex_result_directory: Path) -> List[Path]:
         )
 
     return stats_json_paths
+
+
+def create_default_sample_lane_metrics(
```

Comment:
```suggestion
def create_sample_lane_metrics(
```
I think the default adds confusion. Is this used to just initiliase a SampleLaneMetrics for undetermined reads? If so this should be clarified I think.